src/components/PoolingWidget/index.tsx

```diff
@@ -272,6 +291,16 @@ const PoolingInterface: React.FC = () => {
               <SubComponents step={step} {...restProps} />
             </ContentWrapper>
 
+            <LiquidityMessage>
+              <p>
+                Your liquidity is equal to the amount you have deposited into your exchange wallet.
+                <br />{' '}
+                <b>
+                  Be sure to deposit at least one stablecoin on the <a href="/wallet">Balances</a> tab.
```

Comment:
```suggestion
                  Be sure to deposit at least one stablecoin on the <Link to="/wallet">Balances</Link> tab.
```
use router, don't reload page

Reply:
Thanks, will update. I think we should also do it in the deposit message (in the trade form).